### ISSUE TITLE
whitescreen creating map

### ISSUE THREAD

**Issue by DwieDima** [capacitor-community/capacitor-googlemaps-native#131]
**Describe the bug**

**!! This issue is related to the latest build [v2.0.0-alpha.6](https://github.com/DutchConcepts/capacitor-google-maps/releases/tag/v2.0.0-alpha.6) from the fork of @tafelnl !!**

when you create the map, it is not initialized, although this was successful in the logs (see safari screenshot). The Maps API key is valid. 

**To Reproduce**
Install latest version:
`npm i git+https://github.com/DutchConcepts/capacitor-google-maps.git#v2.0.0-alpha.6`

Code to initialize:

```

ngOnInit() {
    this.folder = this.activatedRoute.snapshot.paramMap.get('id');
    setTimeout(() => {
      this.initMap();
    }, 5000);
  }

  public async initMap() {
    await CapacitorGoogleMaps.initialize({
      key: 'MAPS_API_KEY',
      devicePixelRatio: window.devicePixelRatio, // this line is very important
    });

    const element = document.getElementById('container');
    const boundingRect = element.getBoundingClientRect();
    try {
      //@ts-ignore
      const result = await CapacitorGoogleMaps.createMap({
        boundingRect: {
          width: Math.round(boundingRect.width),
          height: Math.round(boundingRect.height),
          x: Math.round(boundingRect.x),
          y: Math.round(boundingRect.y),
        },
      });

      // remove background, so map can be seen
      element.style.background = 'transparent';

      // finally set `data-maps-id` attribute for delegating touch events
      element.setAttribute('data-maps-id', result.googleMap.mapId);

      console.log('success');
    } catch (e) {
      console.log('error', e);
    }
  }
```

**Screenshots**

***logs inside safari:***
![Bildschirmfoto 2022-02-04 um 13 51 16](https://user-images.githubusercontent.com/26873275/152545176-c830eea3-aeec-46a7-85eb-b58def9c5f04.png)

***logs inside xcode:***
![Bildschirmfoto 2022-02-04 um 15 22 27](https://user-images.githubusercontent.com/26873275/152545253-961aa055-79b2-4fbd-a328-b17f421b9505.png)

***result:***
![RPReplay_Final1643979170](https://user-images.githubusercontent.com/26873275/152545319-7e7291cc-6636-4876-9630-546bd51cb4b3.gif)

**Additional context**
- iPhone 11 Pro IOS 15.2
- XCode Version 13.2.1
- MacBook Pro M1 Monterey 12.1
- Ionic Angular

**Comment by tafelnl** [comment]
Was able to reproduce this indeed. It seems like an iOS only issue. I have it in like 50% of app opens. Seems like the webview background is made opaque again after the map is initialized. So it seems like a race condition or something.

**Comment by va2ron1** [comment]
I have not tested it and I also I'm not following every change of that fork but I think that the [setupWebView](https://github.com/DutchConcepts/capacitor-google-maps/blob/0d96c3fb8488c62c90ca5314a42586a86549ac4d/ios/Plugin/Plugin.swift#L230) is wrong it should be the webView from the bridge.

```
func setupWebView() {
        self.bridge?.webView?.isOpaque = false
        self.bridge?.webView?.backgroundColor = .clear
        self.bridge?.webView?.scrollView.backgroundColor = .clear
        self.bridge?.webView?.scrollView.isOpaque = false
}
```

**Comment by tafelnl** [comment]
Could you try the hotfix in this branch: https://github.com/DutchConcepts/capacitor-google-maps/tree/hotfix/ios-setupWebView

**Comment by DwieDima** [comment]
still got whitescreen:

![image](https://user-images.githubusercontent.com/26873275/153025823-b54c7234-3fad-4f2a-8343-172c53a667f7.png)

**Comment by tafelnl** [comment]
I cannot reproduce this. With the fix above it works every time for me. Could you share a reproduction repo?

**Comment by va2ron1** [comment]
@tafelnl the problem is with Ionic Framework, if the map wrap in a ion-content it need to set background clear too.

Check this fix, it wont work with modals (I also have a fix for modals but is not there) (https://github.com/capacitor-community/capacitor-googlemaps-native/issues/84#issuecomment-1011397977)

**Comment by DwieDima** [comment]
yes i'm using the ionic framework and the map is renderes inside `ion-content`.
@tafelnl here's the repo:

https://github.com/DwieDima/capacitor-maps-example

**Comment by va2ron1** [comment]
@DwieDima set this in the [folder.page.scss](https://github.com/DwieDima/capacitor-maps-example/blob/main/src/app/folder/folder.page.scss), is a workaround to solved your problem.
```
ion-content {
  --background: transparent;
}
```
@tafelnl this [branch](https://github.com/DutchConcepts/capacitor-google-maps/tree/hotfix/ios-setupWebView) need a lot more work, I uploaded my changes that I did in the main repo so you can take it as a reference to make the fixes. https://github.com/va2ron1/capacitor-googlemaps-native

**Comment by tafelnl** [comment]
This is out of scope for this plugin. Every app and framework has different CSS. We will of course not be able to implement this for every possible scenario out there. Every developer should handle this himself. Of course this should be mentioned in the docs. But it is definitely not a bug or missing feature. The capacitor barcode scanner for example , wields the same principle

**Comment by DwieDima** [comment]
@va2ron1 thank you, your snippet fixed it!
@tafelnl now i found another issue: #132
